federatedscope/core/configs/cfg_data.py

```diff
@@ -23,6 +23,7 @@ def extend_data_cfg(cfg):
     cfg.data.drop_last = False
     cfg.data.sizes = [10, 5]
     cfg.data.shuffle = True
+    cfg.data.server_holds_all = False  # whether the server (client[0]) holds all data, useful in global evaluation case
```

Comment:
It is confusing to claim server = client[0]

Reply:
modified accordingly